static/app/components/projects/useProjectCreationAccess.spec.tsx

```diff
@@ -1,9 +1,9 @@
-const {useProjectCreationAccess} = require('./useProjectCreationAccess');
```

Comment:
woah, weird to see this

Reply:
Yeah, I was equally surprised :D

Reply:
Are there any more in the codebase we can get rid of?

Reply:
No idea, trying to see as I go :D